lib/Storage/CacheableFlysystemAdapter.php

```diff
@@ -34,17 +36,28 @@
  */
 abstract class CacheableFlysystemAdapter extends FlysystemStorageAdapter {
 	/**
-     * This property is used to check whether the storage is case insensitive or not
-     * @var boolean
-     */
-    protected $isCaseInsensitiveStorage = false;
+	 * This property is used to check whether the storage is case insensitive or not
+	 * @var boolean
+	 */
+	protected $isCaseInsensitiveStorage = false;
```

Comment:
tabs vs. spaces -> please unify

Reply:
Okay let me correct this

Reply:
@patrickjahns I checked in the editor it is indented using Tabs only
the diff is because previously this block was indented using tab size: 2 whereas the whole file was indented using tab size: 4